app/css/main.css

```diff
@@ -90,6 +90,48 @@ pre {
   box-sizing: border-box;
 }
 
+/* Logo lockup */
+#logo-lockup {
+  margin-bottom: -24px;
+  white-space: nowrap;
+}
+#logo-lockup::before {
+  width: 72px;
+  height: 52px;
+  background-image: url(../images/logos/p-logo.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: 50% 50%;
+  display: inline-block;
+  margin-right: 16px;
+  content: '';
+}
+#logo-lockup::after {
+  display: none;
+}
+
+
+/* blog styling */
```

Comment:
Blog styles should be moved to templates/base-blog.html

Reply:
Nevermind, templates/base-blog.html is for a single blog post. Disregard this comment.

Reply:
these styling are shared among base-blog.html & /blog, that's why it's moved to main.css